runtime/doc/options.txt

```diff
@@ -6762,8 +6776,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	displayed when 'verbosefile' is set.
 
 						*'viewdir'* *'vdir'*
-'viewdir' 'vdir'	string	(default for MS-DOS and Win32: "$VIM/vimfiles/view",
-				 for Unix: "~/.vim/view")
+'viewdir' 'vdir'	string	(default: "$XDG_DATA_HOME/nvim/view")
```

Comment:
@kopischke's proposal makes sense to me, so :+1: 

I would favor using `$XDG_DATA_HOME` for `&viewdir`, because of the fact that views are user created.